src/S3/Crypto/S3EncryptionMultipartUploader.php

```diff
@@ -19,7 +19,12 @@
  */
 class S3EncryptionMultipartUploader extends MultipartUploader
 {
-    use EncryptionTrait, CipherBuilderTrait, CryptoParamsTrait;
+    use CipherBuilderTrait;
+    use CryptoParamsTrait;
+    use EncryptionTrait;
+    use UserAgentTrait;
+
+    const VERSION = '1';
```

Comment:
I'd rather see this be named something like "CRYPTOGRAPHY_VERSION" so we know what it's referring to without having to find where it's used later in the code.